### ISSUE TITLE
[a11y] 9.1.4.3 - Sufficient text contrast

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5384]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------

## General notes
- **Apps-Menu:** the link "Dateien" has insufficient contrast on mouse hover between background and foreground color - ca. 2,6:1. Also the underscore won’t fix the issue as there is the same contrast problem.
- **User menu:** the links "Profil" and "Einstellungen" don’t have sufficient contrast on mouse hover between background and forground color: (#FFFFFF) / about 2,6:1.

## Status: partly fulfilled
- **Page 1 (Alle Dateien)**
The buttons "Neu", "Hierher verschieben" and "Hier einfügen" don’t have sufficient contrast on mouse hover between foreground (#FFFFFF)- and background color- about 2,6:1. Similar issue with the links of the breadcrumb navigation ("Verschieben" and "Kopieren" view), public links (URLs) and buttons "Umbenennen" ("Umbenennen"), "Teilen" ("Personen hinzufügen"), "Erstellen" ("neuen  öffentlichen Link erstellen").
See general notes

- **Page 2 (Mit anderen geteilten Dateien)**
See general notes
- **Page 3 (Gelöschte Dateien)**
See general notes
- **Page 4 (Konto)**
See general notes

**Comment by pascalwengerter** [comment]
Superseded by the more recent #10726
